### DISCUSSION TITLE
Created draft dual core NUCLEO-H755ZI-Q

### DISCUSSION THREAD

**Discussion by lonejack** [cesanta/mongoose#2564] [Ask questions, get answers] [answered]
Hi, I've made a draft version on a nucleo dual core board.
Would like to publish on github.
Question about licence: the code from CubeIde is "AS_IS" but I want to specify that mongoose files follow other (your) rules...
Probably someone faced this item. 
A trace project would be helpful...
C.

**Comment by scaprile** [comment, marked as answer]
You have several examples to see in the examples section. Did you notice those whose name has 'cube' in it ?
You don't specify a license. Vendor code carries what vendor code has. What you contribute is owned by Cesanta 
From the README:
## Contributions

Contributions are welcome! Please follow the guidelines below:

- Sign [Cesanta CLA](https://cesanta.com/cla.html) and send GitHub pull request
- Make sure that PRs have only one commit, and deal with one issue only
